### PR TITLE
CLC-6066, less brittle HubEdos module via safe response handling

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-6066
https://jira.berkeley.edu/browse/SISRP-11933

This brings a new home for reusable code and, hopefully, a bit more resiliency to the HubEdos module.

### CODE REVIEW

File: app/models/hub_edos/response_handler.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module HubEdos
+  module ResponseHandler
+
+    def get_students(response)
+      container = response['studentResponse']
```
Comment (paulkerschen) [inline app/models/hub_edos/response_handler.rb]: @johncrossman Do we also need a nil check on `response` itself? It's a little hard to tell from the logs, but I think a nil parsed response caused the blowup in https://jira.berkeley.edu/browse/SISRP-11933.

Comment (johncrossman) [inline app/models/hub_edos/response_handler.rb]: One can never be too safe, @paulkerschen   Comin' right up!

**Comment by paulkerschen** [comment]
One comment above; a welcome improvement overall.

**Comment by paulkerschen** [comment]
Happy T. Thanks @johncrossman!

**Pull request merged**